src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/UploadDocumentHelper.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.helpers;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.util.CollectionUtils;
+import uk.gov.hmcts.et.common.model.ccd.CaseData;
+import uk.gov.hmcts.et.common.model.ccd.CaseDetails;
+import uk.gov.hmcts.et.common.model.ccd.items.DocumentTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.types.ClaimantIndType;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+
+import static com.google.common.base.Strings.isNullOrEmpty;
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.REJECTED_STATE;
+
+/**
+ * Helper methods for the Upload Documents event.
+ */
+@Slf4j
+public final class UploadDocumentHelper {
+
+    private UploadDocumentHelper() {
+        // Access through static methods
+    }
+
+    /**
+     * Returns true if the case is rejected, has a rejection of claim document and has not got an email sent flag set.
+     * @param caseDetails details of the case
+     */
+    public static Boolean shouldSendRejectionEmail(CaseDetails caseDetails) {
```

Comment:
if it is not necessary you could use primitive instead of object (I mean boolean if you dont need something special)

Reply:
Good catch, have changed